README.md

```diff
@@ -1,16 +1,16 @@
 # opentracing go-redis
 
-[OpenTracing](http://opentracing.io/) instrumentation for [go-redis](https://github.com/go-redis/redis).
+[OpenTracing](http://opentracing.io/) instrumentation for [go-redis](https://github.com/go-redis/redis) with support of `process` and `processPipeline` commands.
 
 ## Install
 
 ```
-go get -u github.com/smacker/opentracing-go-redis
+go get -u github.com/ticketmaster/opentracing-go-redis
```

Comment:
Let's keep the original repo, that wont work well when the PR is accepted in that repo

Reply:
Allright